pytest_catchlog.py

```diff
@@ -11,6 +12,29 @@
 
 __version__ = '1.2.0'
 
+TERMINAL = py.io.TerminalWriter(sys.stderr)  # pylint: disable=no-member
+CONSOLEHANDLER = logging.StreamHandler(TERMINAL)
+# Add the handler to logging
+logging.root.addHandler(CONSOLEHANDLER)
+# The root logging should have the lowest logging level to allow all messages
+# to be "passed" to the handlers
+logging.root.setLevel(1)
+
+
+class Hooks(object):  # pylint: disable=too-few-public-methods
+    """
+    Class to add new hooks to pytest
+    """
+
+    def pytest_register_logging_levels(self):
```

Comment:
I'd like to avoid guessing or getting close. The logging module should give use the available logging levels, though, libraries which are late imported could add log levels after the plugin has had a chance to translate verbosity to log level...